### PR TITLE
Feature/hooks

### PR DESCRIPTION
**Pull request by Gnwin**
## Math Magicians: Refactor with hooks

In this milestone, the logic of the application was converted to stateful functional components with react hooks without without any errors. All requirements for this milestone have been met according to this document:

https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_refactor_with_hooks.md

### CODE REVIEW

File: src/reactjs/components/Calculator.js [hunk 1]
```diff
@@ -56,7 +56,6 @@ class Calculator extends React.Component {
         operation: '',
       });
     } else {
-      calcData = calculate(this.state, buttonContent);
```
Comment (GzimAsani) [inline src/reactjs/components/Calculator.js:59]: - [x] The calculator component should also be refactored similar to app.js

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ A website with a calculator
 To get a local copy up and running follow this simple step:
 
 - Clone repository to your local machine: `www.github.com/gnwin/math-magicians.git`
-- Run `git checkout feature/events` to switch to develop branch
+- Run `git checkout feature/hooks` to switch to develop branch
```
Comment (GzimAsani) [inline README.md]: - [x] You have some merging conflict, please resolve it. I think the conflict comes from readme file

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @Gnwin     
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your code organization is good.
 
 ## Required Changes 
 - Please explain better(in more detail) the steps that a user should take in order to clone and run your project




 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @Gnwin,


## Wow, you did it 🎉
Brilliant
## Thank you for refactoring your Calculator component, and for solving the readme issue, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gnwin** [comment]
Thanks @Meltrust

**Pull request merged**